### PR TITLE
Route for refactor

### PR DESCRIPTION
**Pull request by GiantRobots**
## Checklist

1. Closes 

2. How was this tested:

3. Any docs updates needed?

### CODE REVIEW

**Comment by stevekinney** [comment]
Can we conform to the same way we do [the client-side `routeFor`](https://github.com/temporalio/ui/blob/main/src/lib/utilities/route-for.ts)?

File: src/lib/utilities/route-for-api.ts [hunk 1]
```diff
@@ -1,48 +1,67 @@
+import urlcat from 'urlcat';
+
 let base = (import.meta.env?.VITE_API as string) ?? process.env.VITE_API;
-if (base.endsWith('/')) base = base.slice(0, -1);
+base = `${base}/api/v1/`;
```
Comment (feedmeapples) [inline src/lib/utilities/route-for-api.ts:4]: could you add few tests for relative paths when `VITE_API` is set to empty string or `/`

**Review by feedmeapples — CHANGES_REQUESTED** [review]
